test/lib/redis-store-zlib-spec.js

```diff
@@ -0,0 +1,372 @@
+var config = require('../config.json');
+var redisStore = require('../../index');
+var sinon = require('sinon');
+var assert = require('assert');
+var zlib = require('zlib');
+
+var redisGzipCache;
+var customRedisGzipCache;
+var testJson;
+
+describe('Gzip Tests', function () {
+
+  before(function () {
+    redisGzipCache = require('cache-manager').caching({
+      store: redisStore,
+      host: config.redis.host,
+      port: config.redis.port,
+      auth_pass: config.redis.auth_pass,
+      db: config.redis.db,
+      ttl: config.redis.ttl,
+      gzip: true
```

Comment:
That is a good idea, will avoid having to deal with a lot of options later on and open the road to the compression pipelining :smile: